package.json

```diff
@@ -11,11 +11,13 @@
   ],
   "scripts": {
     "build": "fedx-scripts webpack",
+    "build:with-theme": "paragon install-theme && npm run build",
     "i18n_extract": "BABEL_ENV=i18n fedx-scripts babel src --quiet > /dev/null",
     "lint": "fedx-scripts eslint --ext .js --ext .jsx .",
     "lint:fix": "fedx-scripts eslint --fix --ext .js --ext .jsx .",
     "snapshot": "fedx-scripts jest --updateSnapshot",
     "start": "fedx-scripts webpack-dev-server --progress",
+    "start:with-theme": "paragon install-theme --brand=@edx/brand-edx.org@latest && npm start",
```

Comment:
Can this be made generic, somehow?  Otherwise, the command would warrant being called `start:with-edx-theme`... and in that case, including it here would be debatable.

Reply:
@arbrandes Ah, yes. Let me mark this PR as a draft as it's still in flux a bit. The `paragon install-theme` command does now accept a `--brand` argument for more programmatic usage of the CLI tool (e.g., in build+deploy processes, if needed). But, we do not intend to include `--brand` as part of the NPM scripts in MFEs.

Instead, if the `--brand` argument is not passed, it prompts the user in terminal for the name/version of an NPM package to install as the `@edx/brand` alias instead.

We also heard some feedback recently, where a third usage pattern for this command might be a Git-ignored config file where an engineer could specify something like the following to persist the custom `@edx/brand` theme selection without requiring manual input each time.

```js
// paragon-cli.config.js, gitignored
{
  'commands': {
    'install-theme': {
      'brand': '@edx/brand-edx.org@latest',
    }
}
```